lib/bolt/node/orch.rb

```diff
@@ -53,6 +53,8 @@ def _run_task(task, _input_method, arguments)
 
       if state == 'finished'
         exit_code = 0
+      elsif state == 'skipped'
+        return Bolt::Result.new('Node was skipped')
```

Comment:
This should probably be a TaskResult with a skipped error. I have an example error here https://github.com/puppetlabs/task-modules/blob/master/site/canary/lib/puppet/functions/canary/skip.rb

Reply:
ok